### PR TITLE
Develop author details page

### LINKED ISSUE #6: Develop author details page
**Brief description of the feature? Please describe.**
In this issue, write story page need to be developed. write story page contains -

- Trending stories (right corner )
- bookmark story 
- total no of shares
- share story
- story details
- author details
- comments

**Links and resources**
[link to the wireframe](http://bit.ly/she-author-page)
**Screenshots for the new feature**
[screenshot](http://bit.ly/she-author-page)

**Additional Information**
NA

### PR DESCRIPTION
**Pull request by vaibhav-sharma-frontend**
Develop author details page i.e. added author-details.html & author-details.css pages along with responsiveness.

Issue Fixed: #6 

**Screenshots:**

Screenshot from 2019-05-25 19-32-05
Screenshot from 2019-05-25 19-32-14
Screenshot from 2019-05-25 19-32-18
Screenshot from 2019-05-25 19-32-21
Screenshot from 2019-05-25 19-32-26
Screenshot from 2019-05-25 19-32-32
Screenshot from 2019-05-25 19-32-43
Screenshot from 2019-05-25 19-32-50
Screenshot from 2019-05-25 19-32-54
Screenshot from 2019-05-25 19-33-00
Screenshot from 2019-05-25 19-33-12
Screenshot from 2019-05-25 19-43-27
Screenshot from 2019-05-25 19-43-31
Screenshot from 2019-05-25 19-43-34
Screenshot from 2019-05-25 19-43-37
Screenshot from 2019-05-25 19-43-43
Screenshot from 2019-05-25 19-43-47
Screenshot from 2019-05-25 19-43-49

### CODE REVIEW

File: pages/author-details.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                </li>
+            </ul>  
+        </div>
+    </nav>
+    
+    <!-- Page Content -->
+    <div class="container mt-5 mb-5">
+        <div class="row">
+            <div class="col-lg-7 col-md-12">
+                <h2 class="row writer-story-heading">She finds the biggest potential for sustainability, from being a passionate Dancer to Motivational Speaker</h2>
```
Comment (kindacoder) [inline pages/author-details.html:55]: We can work on this one ? It's not looking good. What do you think ?

File: pages/author-details.html [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+                    <p>Currently, in her lifestyle, she finds the biggest potential for sustainability in her choice of working for development both on a professional and a personal level. Be it becoming vegetarian, walking to get around, or not using plastics or air conditioners, she tries her best to reduce her carbon footprint.</p>
+                    <p>However, it is still a long journey, she says. “Even making small changes and rethinking the ways our daily lives function can help create a movement towards a more sustainable lifestyle. It is equally important to incorporate empathy into our relationships, to share stories with each other and to take the time to listen, especially when so many of us cross paths every day but don’t realize that there are ways to help each other.”She is a strong propagator of stories and likes conducting qualitative research quantitative evidence needs to be supported with anecdotes to have deeper insights. At the age of 24, she has already authored several blog posts, given talks at various international conferences and co-edited the first edition of the Youth Solutions Report.</p>
+                    <h4>Message to Women: </h4>
```
Comment (kindacoder) [inline pages/author-details.html:104]: **Message to women** This heading should come after a divider/horizontal line or something like that.

File: pages/author-details.html [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+                    <p>Her belief that we can work together for the cause of reducing poverty regardless of race, culture, sex, occupation, or any geographical boundaries, and this motivates her to live sustainably in her own life. She also believes there is a place for performance arts to create impact, and that everyone can use their unique abilities to produce something good. Dance is still an important part of her life that was earlier a medium to voice her opinion, create awareness, and express her concerns against social inequality.</p>
+                    <img class="story-image" src="http://sheendeavor.000webhostapp.com//uploads/b281baadf247f6343753635671292ccfd54fb154.jpg" alt="Kanika Joshi Image">
+                    <p>Her parents and the generations before them originally came from Uttarakhand, the birth town of the Chipko movement located at the footsteps of the Himalayas.  Her family carried on the traditions of its homeland by always staying environmentally conscious.</p>
+                    <h4>Contributions:</h4>
```
Comment (kindacoder) [inline pages/author-details.html:101]: All the heading should be divided. They are looking like a paragraph.It also needs some improvement.

File: pages/author-details.html (hunk 1 above)
Comment (kindacoder) [inline pages/author-details.html:55]: This should be in the centre with some modification in style. What do you think ?

**Review by kindacoder — COMMENTED** [review]
I have reviewed, please do the modifications as requested. :+1:

File: pages/author-details.html (hunk 1 above)
Comment (vaibhav-sharma-frontend) [reply pages/author-details.html:55]: the design I think look a bit odd I think

**Comment by vaibhav-sharma-frontend** [comment]
@kindacoder I will try to give it a new look wait till tonight

**Review by prithaupadhyay — APPROVED** [review]
LGTM 👍

**Pull request merged**